docs/source/tags/number.md

```diff
@@ -19,11 +19,12 @@ Use with the following data types: audio, image, HTML, paragraphs, text, time se
 | [min] | <code>number</code> |  | Minimum number value |
 | [max] | <code>number</code> |  | Maximum number value |
 | [step] | <code>number</code> | <code>1</code> | Step for value increment/decrement |
-| [defaultValue] | <code>number</code> |  | Default number value |
-| hotkey | <code>string</code> |  | Hotkey for increasing number value |
+| [defaultValue] | <code>number</code> |  | Default number value; will be added automaticaly to result for required fields |
+| [hotkey] | <code>string</code> |  | Hotkey for increasing number value |
```

Comment:
Should line #17 and #18 also have `[name]` and `[toName]` instead of `name` and `toName`?

Reply:
they are required